### ISSUE TITLE
OSError: Could not load shared object file: llvmlite.dll

### ISSUE THREAD

**Issue by crusaderky** [dask/dask#8549]
There's an upstream bug in llvmlite on Windows that will cause the process scheduler to randomly fail:

https://github.com/numba/llvmlite/issues/808

**Comment by crusaderky** [comment]
Interesting - I'm seeing it fail *always* on test_set_index_consistent_divisions, *never* on scheduler="processes", and exclusively on Python 3.7/Windows. Not sure about the reason.

**Comment by rjzamora** [comment]
I am getting Python 3.7/Windows failures in several PRs (where no other CI is failing). `test_set_index_consistent_divisions` always fails, but the other failures vary from run to run. For example, #8551 has 3 failures and it is just a fake docstring change.

**Comment by scharlottej13** [comment]
@crusaderky can we close this issue?

**Comment by crusaderky** [comment]
@scharlottej13 no, the issue is still there we just xfailed the test for the time being.
